src/VisualStudio/Core/Def/Guids.cs

```diff
@@ -127,6 +127,10 @@ internal static class Guids
         public const string StackTraceExplorerCommandIdString = "FB190424-4DFF-43DB-8CCA-E32D1CE8A5CA";
         public static readonly Guid StackTraceExplorerCommandId = new(StackTraceExplorerCommandIdString);
 
+        // Create new guids and have them be available as static strings
+        public const string SampleToolWindowIdString = "F85D2A49-72B5-4B38-ABED-6F177D88F81D";
+        public static readonly Guid SampleToolWindowId = new(SampleToolWindowIdString);
```

Comment:
Will these be removed when we move to using the VS Outline toolwindow? If not, we would want to use more meaningful names.

Reply:
I believe they will be removed once we move to using the VS Outline toolwindow, otherwise I will happily change the names :)